src/Conference/Admin/Settings.php

```diff
@@ -233,4 +243,117 @@ public function options_page_html() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Renders the Order Sponsor Levels admin page.
+	 *
+	 * @since TBD
+	 */
+	public function render_order_sponsor_levels() {
+	    $this->render_order_levels('wpcsp_conference_sponsor_level_order', Plugin::SPONSOR_LEVEL_TAXONOMY, _x('Change the order of sponsor levels displayed in the sponsors page template.', 'Directions for ordering of sponsor levels in the settings.', 'wp-conference-schedule'));
+	}
+
+	/**
+	 * Renders the Order Speaker Levels admin page.
+	 *
+	 * @since TBD
+	 */
+	public function render_order_speaker_levels() {
+	    $this->render_order_levels('wpcsp_speaker_level_order', Plugin::GROUP_TAXONOMY, _x('Change the order of speaker groups displayed in the speaker page template.', 'Directions for ordering of speaker groups in the settings.', 'wp-conference-schedule'));
+	}
+
+	/**
+	 * General method to render the order levels.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $option    The option key to fetch from the database.
+	 * @param string $taxonomy  The taxonomy to fetch terms for.
+	 * @param string $message   The instructional message to display.
+	 */
+	private function render_order_levels(string $option, string $taxonomy, string $message) {
+	    if ( ! isset( $_REQUEST['updated'] ) ) {
+	        $_REQUEST['updated'] = false;
+	    }
+
+	    $levels = $this->get_sponsor_levels($option, $taxonomy);
+	    if( empty($levels) ) {
+	        return;
+	    }
+	    ?>
+	    <div class="description sponsor-order-instructions">
+	        <?php echo $message; ?>
+	    </div>
+	    <ul class="sponsor-order">
+	    <?php foreach ( $levels as $term ) : ?>
+	        <li class="level">
+	            <input type="hidden" class="level-id" name="<?php echo $option; ?>[]" value="<?php echo esc_attr( $term->term_id ); ?>" />
+	            <?php echo esc_html( $term->name ); ?>
+	        </li>
+	    <?php endforeach; ?>
+	    </ul>
+	    <?php
+	}
+
+	/**
+	 * Validates the sponsor options ensuring they are an array of integers.
+	 *
+	 * @since TBD
+	 *
+	 * @param array|null $input The input options array.
+	 *
+	 * @return array|null Sanitized array of integers or null.
+	 */
+	public function validate_sponsor_options(?array $input): ?array {
+	    if (!is_array($input)) {
+	        return null;
+	    }
+
+	    return array_values(array_map('intval', $input));
```

Comment:
```suggestion
	public function validate_sponsor_options( ?array $input ): ?array {
	    if ( ! is_array( $input ) ) {
	        return null;
	    }

	    return array_values( array_map( 'intval', $input ) );
```

Reply:
I fixed the spacing. Let me know if there is something else. Thanks.